deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobIT.java

```diff
@@ -101,6 +102,12 @@ public void init() throws Exception {
 
 		techmdDir = new File(depositDir, TECHMD_DIR);
 		techmdDir.mkdir();
+		
+		File modsFolder = job.getDescriptionDir();
```

Comment:
There's a lot of stuff going on in the ingestWorkObjectTest method.  How would you feel about making another test that is dedicated to testing addDescription?  It'd just be a very slimmed down version of the same thing you already did, with or without using a Work, up to you.  

Also explicitly test that another object that shouldn't get mods doesn't get it.

Also, if workPidForMods and the modsFolder are only used for one test you should probably just put them into the test that uses them.